recirq/qml_lfe/run_config.py

```diff
@@ -59,6 +61,30 @@ def execute_batch(
     return [x[0] for x in res]
 
 
+def execute_sweep(
+    circuit: cirq.Circuit, params: List[Dict[str, float]], use_engine: bool
+) -> List[cirq.Result]:
+    """Use either simulator or engine to run a one rep sweep on a circuit.
+
+    Args:
+        circuit: the `cirq.Circuit` with symbols to execute.
+        params: The list of parameters to place in the symbols.
+        use_engine: Whether or not to make use of quantum engine and the
+            weber processor. Note this requires the GOOGLE_CLOUD_PROJECT
+            environment variable to be set, along with the required cloud
+            permissions.
+    Returns:
+        List of results to execution.
+    """
+    runner = cirq.Simulator()
```

Comment:
not: Maybe move this to an "else" block to avoid instantiation in the engine case?

Reply:
I had figured empty simulator instantiation was pretty light, but I've moved to an else block just to be safe.